docs/validators/key-management/multikey-nodes.md

```diff
@@ -0,0 +1,123 @@
+---
+id: multikey-nodes
+title: Multikey nodes management
+---
+
+[comment]: # (mx-abstract)
+This page contains information about how to manage multiple keys on an observing squad.
+
+[comment]: # (mx-context-auto)
+## Multikey architecture overview
+
+At the mainnet launch, and up until the release candidate RC/v1.6.0 becomes mainstream, each node could have managed only
+one key. So the relationship between the nodes and keys is `1:1`, the node "following" the shard where the managed key is assigned.
+The main idea behind the multikey feature is that a set of keys can be managed by an observing squad set. The same set of keys
```

Comment:
rephrased

Reply:
Since an observer already does the full validation of every block in its own shard, and the only missing operation, as opposed to a validator being its selection and voting during consensus, it becomes easy to adapt such an observer node to manage a group of validator keys, and whenever one of those keys is selected for the consensus, the observer can propose and/or validate and vote for the proposed block on behalf of the keys from the consensus group it manages.

=>

Since an observer already performs block validation in its own shard, it can be easily used to manage a group of validator keys and propose or validate blocks on behalf of the keys it possesses.

Reply:
re-rephrased